### PR TITLE
Minor sample notebook cleanup

### PR DESCRIPTION
**Pull request by amygdala**
Fixed some typos, bumped the sdk version, and cleared the cell output, as I thought that it might be confusing (e.g. the output included links to pipeline runs generated during previous tests).

Btw: what are those "# See screenshot N" comments for?  Should they be removed too?

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/164)

### CODE REVIEW

**Comment by amygdala** [comment]
Yeah, I did remove the cell outputs -- I was thinking that the links to the UI (which of course won't work for the users) might be confusing.  Potentially I could just clear those cell outputs with the links and keep everything else?  Or maybe it's not a big deal? Up to you.

**Comment by qimingj** [comment]
I think it is not a big deal for the dead links, but it is important to show user what the expected outputs are. Many users won't run the notebook but are just readers.

**Comment by amygdala** [comment]
okay, that sounds fine.  My second push reverted keeping your cell inputs, so the PR is now just the typo fixes & version bump.
(Though...I see some merge conflict has now appeared, I'll look at it tomorrow).

**Comment by amygdala** [comment]
okay -- I don't know if this PR is worth it, as the bulk of the changes I originally made keep getting pushed by other PRs.  Up to you.

File: samples/notebooks/KubeFlow Pipeline Using TFX OSS Components.ipynb [hunk 1]
```diff
@@ -31,7 +31,7 @@
    "outputs": [],
    "source": [
     "# Install Pipeline SDK\n",
-    "!pip3 install https://storage.googleapis.com/ml-pipeline/release/0.1.1/kfp.tar.gz --upgrade\n",
+    "!pip3 install https://storage.googleapis.com/ml-pipeline/release/0.1.2/kfp.tar.gz --upgrade\n",
```
Comment (gaoning777) [inline samples/notebooks/KubeFlow Pipeline Using TFX OSS Components.ipynb]: I guess this PR has been waiting for too long and the version was actually bumped previously.

**Comment by amygdala** [comment]
I'll close this... at the time it was a useful fix, but the 'tests fail on external PRs' issue made it languish too long.

**Pull request closed without merging**